### PR TITLE
Feat/convert currency

### PR DESCRIPTION
**Pull request by RaceOsuna**
What I did:

- Implemented 3rd party api to convert currency
- wrote html for currency conversion container
- added catches and error handling for api calls
- swapped out api end points

Did this break anything?
- [x] No
- [ ] Yes

Type of change

- [x] New feature (non-breaking change which adds functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] Refactor(DRY-ing up/ reorganizing code, etc.)
- [ ] Super small fix (Corrected a typo, removed a comment, etc.)
- [ ] Skip all the other stuff and briefly explain the fix.

Checklist:
- [ ] If this code needs to be tested, all tests are passing
- [x] The code runs locally
- [ ] There are comments where clarification/ organization is needed
- [x] The code is DRY. If it's not, I tried my best
- [x] I reviewed my code before pushing

What's next?

-???

### CODE REVIEW

File: src/apiCalls.js [hunk 1]
```diff
@@ -15,14 +37,24 @@ usersData = users.users
...
+  })
+  .catch(error => alert(`${error.message}`));
 
 export {
     recipeData,
     usersData,
     ingredientsData,
-    user
+    user,
 }
```
Comment (AlexaVanSchaardenburg) [inline src/apiCalls.js:60]: Can we remove the comma on line 27?

**Review by AlexaVanSchaardenburg — APPROVED** [review]
This looks good! So impressed with how quickly you knocked this out!

**Review by rachelsoae — APPROVED** [review]

File: src/apiCalls.js (hunk 1 above)
Comment (rachelsoae) [reply src/apiCalls.js:60]: Additional exports have been added below this line, so this is now resolved :)

**Comment by rachelsoae** [comment]
Resolves issues #41

**Pull request merged**